datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -18,6 +18,73 @@
 
 # match integration's version within the __about__.py module
 VERSION = re.compile(r'__version__ *= *(?:[\'"])(.+?)(?:[\'"])')
+DOGWEB_JSON_DASHBOARDS = (
```

Comment:
How do we plan to keep these lists in sync?

Should we add a small assertion in `has_dashboard()` so that we raise an error if the check is marked as being in dogweb but a dashboard was found in `assets/dashboards`?

Reply:
We don't add dashboards in dogweb any more. I think we'd only be updating these lists if we have plans to migrate these dashboards to `-core`

Reply:
Yes, also for this I don't care where the dashboard is only if it exists.

Reply:
for example I just realised I added rabbitmq to both sets, it's still ok